<?xml version="1.0"?>

<launch>

  <arg name="node_name" default="sim_0"/>
  <arg name="port" default="19997"/>

  <!--Load parameters-->
  <rosparam command="load" file="$(find sfm_planner)/config/test_conductor.yaml" />

  <group ns="$(arg node_name)">
      <!--Run Test Conductor-->
      <node name="test_conductor" pkg="sfm_planner" type="test_conductor.py" args="$(arg node_name) $(arg port)" />
  </group>

</launch>
